src/screens/explore/About/index.tsx

```diff
@@ -1,30 +1,59 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Image, Text, TouchableOpacity, View, ScrollView } from 'react-native';
 import { Feather } from '@expo/vector-icons';
+import { connect } from 'react-redux';
 import get from 'lodash/get';
 import { navigate } from '../../../navigationRef';
+import { Context as AuthContext } from '../../../context/AuthContext';
 import styles from './styles';
 import { WHITE } from '../../../styles/colors';
 import { ICON } from '../../../styles/metrics';
+import { NavigationType } from '../../../types/NavigationType';
 import { ProgramType } from '../../../types/ProgramType';
+import Button from '../../../components/form/Button';
+import Courses from '../../../api/courses';
+import { getLoggedUserId } from '../../../store/main/selectors';
 
 interface AboutProps {
   route: { params: { program: ProgramType } },
+  navigation: NavigationType,
+  loggedUserId: string,
 }
 
-const About = ({ route }: AboutProps) => {
+const About = ({ route, navigation, loggedUserId }: AboutProps) => {
   const { program } = route.params;
+  const { signOut } = useContext(AuthContext);
 
   const programImage = get(program, 'image.link') || '';
   const programName = get(program, 'name') || '';
   const programDescription = get(program, 'description') || '';
   const source = programImage
     ? { uri: programImage }
     : require('../../../../assets/images/authentication_background_image.jpg');
+  const subProgram = program.subPrograms ? program.subPrograms[0] : null;
```

Comment:
ici on ne recupere que les sous-programmes elearning depuis le back ?

Reply:
Normalement oui avec le `match: { _id: { $in: subPrograms } },` dans le populate

Reply:
j'ai l'impression qu'ici du coup on a toujours `subPrograms` non ?

Reply:
Ici oui mais comme on utilise ProgramType et que parfois dans les objets program il n'y a pas de sous program, on le met en optionnel et eslint nous met une erreur ici. Je pensais m'en occuper dans une prochaine pr en faisant la refacto pour les interfaces.